src/app/api/book/[symbol1]/[symbol2]/[hops]/[limit]/route.ts

```diff
@@ -0,0 +1,53 @@
+import { NextResponse } from 'next/server';
+import { DexQueryServiceClient } from '@/old/utils/protos/services/dex/dex-query-service-client';
+import { DirectedTradingPair } from '@penumbra-zone/protobuf/penumbra/core/component/dex/v1/dex_pb';
+import { Metadata } from '@penumbra-zone/protobuf/penumbra/core/asset/v1/asset_pb';
+import { ChainRegistryClient } from '@penumbra-labs/registry';
+
+const chainRegistryClient = new ChainRegistryClient();
+const querier = new DexQueryServiceClient({
+  grpcEndpoint: process.env['PENUMBRA_GRPC_ENDPOINT'] ?? '',
+});
+
+interface Params {
+  symbol1: string;
+  symbol2: string;
+  hops: string;
+  limit: string;
+}
```

Comment:
idea: not for this PR, but think trpc would be good for us to use

Reply:
@grod220 @JasonMHasperhoven wouldn't it be easier to not use the `api` directory but a simple server-side fetch within a component? Like

```tsx
export default async function Page() {
  let data = await fetch('https://api.vercel.app/blog')
  let posts = await data.json()
  return (
    <ul>
      {posts.map((post) => (
        <li key={post.id}>{post.title}</li>
      ))}
    </ul>
  )
}
``` 

Instead of the `fetch` from my example, we could write `useAsksBids()` from the API made by Jason. It would do all the server-side work while sharing the types eliminating the need in TRPC. Wdyt?